docs/testplan/HA-SmartSwitch-test-plan.md

```diff
@@ -0,0 +1,188 @@
+|-----------------------------------------|----------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|--------------------------------------------------|---------------------------------------------------|
+| Active NPU-to-DPU probe drop -Active    | Verify packet flow when NPU1 to DPU1 link starts dropping probe packets. | • Start  I/O through active side.<br>• Configure the NPU1-to-DPU1 link to drop packets.         | DPU1 becomes non-active, DPU2 becomes standalone. | T2 receives packets with 1 allowed disruption.  |
+| Active NPU-to-DPU probe drop -Standby   | Verify packet flow when NPU1 to DPU1 link starts dropping probe packets. | • Start  I/O through standby side.<br>• Configure the NPU1-to-DPU1 link to drop packets.        | DPU1 becomes non-active, DPU2 becomes standalone. | T2 receives packets with 1 allowed disruption.  |
+| Standby NPU-to-DPU probe drop – Active  | Verify packet flow when NPU2 to DPU2 link starts dropping probe packets. | • Start  I/O through active side.<br>• Configure the NPU2-to-DPU2 link to drop packets.         | DPU1 becomes standalone, DPU2 is anything but active. | T2 receives packets without disruption.          |
+| Standby NPU-to-DPU probe drop – Standby | Verify packet flow when NPU2 to DPU2 link starts dropping probe packets. | • Start  I/O through standby side.<br>• Configure the NPU2-to-DPU2 link to drop packets.       | DPU1 becomes standalone, DPU2 is anything but active. | T2 receives packets without disruption.          |
+| Active T1-T0 link drop – Active         | Verify packet flow when T1-T0 link drop.                      | • Start  I/O through active side.<br>• Configure DPU1 side T1-T0 link drop.                       | DPU1 becomes non-active, DPU2 becomes standalone. | T2 receives packets with 1 allowed disruption.  |
+| Active T1-T0 link drop – Standby        | Verify packet flow when T1-T0 link drop.                      | • Start  I/O through standby side.<br>• Configure DPU1 side T1-T0 link drop.                      | DPU1 becomes non-active, DPU2 becomes standalone. | T2 receives packets with 1 allowed disruption.  |
+| Standby T1-T0 link drop – Active        | Verify packet flow when T1-T0 link drop.                      | • Start  I/O through active side.<br>• Configure DPU2 side T1-T0 link drop.                       | DPU1 becomes standalone, DPU2 is anything but active. | T2 receives packets without disruption.          |
+| Standby T1-T0 link drop - Standby      | Verify packet flow when T1-T0 link drop.                      | • Start  I/O through standby side.<br>• Configure DPU2 side T1-T0 link drop.                      | DPU1 becomes standalone, DPU2 is anything but active. | T2 receives packets without disruption.          |
+
```

Comment:
will be better to provide a ACL example here on how to drop the probe packets.

Reply:
Sample is provided.